cli/src/commonMain/kotlin/OrtServerMain.kt

```diff
@@ -69,7 +69,7 @@ fun main(args: Array<String>) {
 
 class OrtServerMain : SuspendingNoOpCliktCommand(COMMAND_NAME) {
     init {
-        completionOption()
+        completionOption(hidden = true)
```

Comment:
Would you mind adding a [check like in ORT](https://github.com/oss-review-toolkit/ort/blob/f26c647a313a312659768a46220e10d05d289e35/.github/workflows/static-analysis.yml#L35-L59) to ensure that the completions are kept up to date?

Reply:
Because I think you should have updated the completion scripts as part of this PR as well?